client/payment-details/dispute-details/dispute-awaiting-response-details.tsx

```diff
@@ -93,7 +115,7 @@ const DisputeAwaitingResponseDetails: React.FC< Props > = ( {
 					/>
 
 					{ /* Dispute Actions */ }
-					{ showDisputeStepsAndActions && (
+					{
```

Comment:
I'm not sold on splitting the Dispute / Inquiry paths. But agree that it's not easily readable.

I'll have a go at breaking this up into something more manageable.

Reply:
I think this is quite subjective, so feel free to leave as-is!

Reply:
I tried to split it off but it was challenging given the `isModalOpen`, `setModal` and, `onModalClose` that had to be passed on.

I have opted instead for a function that returns the various properties rather than all the ternaries.

Reply:
**This PR works as intended, so I don't think my uncertainties below should hold up this PR being shipped.** This is a tricky subject that I struggle to find the correct answers for (see a very similar discussion in https://github.com/Automattic/woocommerce-payments/pull/7047#discussion_r1326775042).

I do think you've improved the readability of this code by the mapping of the labels etc  – this more clearly separates what is for inquiries and what is for disputes. 

However, I still find this entangled a bit and hard to follow. There are still some `isInquiry` ternaries outside of the modal, e.g. clicking `issue refund`/`accept` will do different things – inquiries probably wants to be a `Button href={orderUrl}` rather than `Button onClick={window.location.orderUrl}`.

I've explored another way to write this in #7339, the pseudocode TLDR is:

```tsx
const ExistingConciseMarkup = () => {
	const actionLabelsEtc = isInquiry ? {
		label: 'Inquiry Label',
	} : {
		label: 'Dispute Label',
	};
	
	// Markup is defined once, but logic is not as easy to follow.
	return (
		<Actions>
			{actionLabelsEtc.label}
		</Actions>
	);
}

// Repeat markup to make it clear what is for inquiries and what is for disputes
const SimpleButRepetitiveMarkup = () => {
	// Markup is defined twice, but logic is straightforward.
	return isInquiry ? (
		<Actions>Inquiry Label</Actions>
	) : (
		<Actions>Dispute Label</Actions>
	)
}
```

I'm unsure if this is _more_ or _less_ readable as a whole than the existing solution.